File: Code-prep/Code-prep/1-LinkedList/LinkedList.java
class Node{
	int data;
	Node next;
	Node(int data){
		this.data=data;
		next=null;
	}
}


class LinkedList{
	Node head;
	
	public LinkedList(){
		head=null;
	}
	
	void add(int data){
		Node n=new Node(data);
		Node temp;
		temp=head;
		if (head==null){
			head=n;
		}
		else{
			while(temp.next!=null){
				temp=temp.next;
			}
			temp.next=n;
		}
	}
	
	void printList(){
		System.out.println("Printing LinkedList");
		Node temp;
		temp=head;
		while(temp!=null){
			System.out.println(temp.data);
			temp=temp.next;
		}
	}
	public static void main (String args[]){
		System.out.println("Linked List Basic Program");
		LinkedList L=new LinkedList();
		L.add(5);
		L.add(3);
		L.add(2);
		L.printList();
		
	} 
}

File: Code-prep/Python/1-LinkedList/LinkedList.py
import os


class Node(object):
    data=0.0
    Next=None
    
    def __init__(self,data):
        self.data=data
        self.next=None
        
    def printNode(self):
        print self.data
    
    
class LinkedList(object):
    '''This class provides insertion in linkedlist'''
    head=None
    
    def insert(self,data):
        temp=Node(data)
        if self.head==None:
            self.head=temp
        else:
            cur=self.head
            while cur.Next!=None:
                cur=cur.Next
            cur.Next=temp
            
    def printLL(self):
        cur=self.head
        while cur!=None:
            cur.printNode()
            cur=cur.Next
            
if __name__=="__main__":
    LL=LinkedList()
    LL.insert(2)
    LL.insert(3)
    LL.printLL()